src/storhaug

```diff
@@ -399,6 +402,7 @@ setup_create_resources()
             config="${HA_NFS_CONF}" \
             state_fs="${HA_NFS_TYPE}" \
             state_mnt="${HA_MNT_DIR}/${HA_NFS_MNT_DIR}" \
+        op monitor on-fail="block" \
```

Comment:
Actually under my test, if we keep the default 'restart', then ganesha process will restart which prevent the VIP form failover.
But it is not always safe to restart ganesha when crashed, in the worst case, ganesha will crash, restarat, crash, restart again... I think it is better to crash directly and failover, then an administrator will have some time to fix the problem. After that, ganesha can restart safely.